pkg/controllers/readiness/wellknown_ready_controller.go

```diff
@@ -151,6 +155,16 @@ func (c *wellKnownReadyController) isWellknownEndpointsReady(authConfig *configv
 		}
 	}
 
+	// if we don't have the min number of masters, this is actually ok, however Clayton has draw a hardline on starting tests as soon as all operators are Available=true
+	// while ignoring progressing=false.  This means that even though no external observer will see a invalid .well-known information,
+	// the tests end up failing when their long lived connections are terminated.  Killing long lived connections is normal and
+	// acceptable for the kube-apiserver to do during a rollout.  However, because we are not allowed to merge code that ensures
+	// a stable kube-apiserver and because rewriting client tests like e2e-cmd is impractical, we are left trying to enforce
```

Comment:
I thought @soltysh was in the process of rewriting e2e-cmd?

Reply:
he is